backend/api/curious/types/index.d.ts

```diff
@@ -1,8 +1,7 @@
 /**
  * This file was auto-generated by openapi-typescript.
- * Some minor changes were then necessary.
+ * Do not make direct changes to the file.
```

Comment:
if you managed to generate the file exactly as is needed, could you update the instructions in the READ.me file?